client/components/CandidateReview/CandidateTestPlanRun/index.jsx

```diff
@@ -467,15 +455,16 @@ const CandidateTestPlanRun = () => {
                                 }
                                 issues={list}
                                 individualTest={true}
-                                githubUrl={generateGithubUrl(
-                                    true,
-                                    index === 0
-                                        ? 'changes-requested'
-                                        : 'feedback',
-                                    null,
-                                    true,
-                                    !differentAuthors ? data.me.username : null
-                                )}
+                                githubUrl={getIssueSearchLink({
+                                    isCandidateReview: true,
+                                    isCandidateReviewChangesRequested:
+                                        index === 0,
+                                    username: data.me.username,
```

Comment:
```suggestion
                                    // username: data.me.username,
```

I think it's safer to not include the username so the `getIssueSearchLink` doesn't parse that into the link being checked. It's safer in the instances where the feedback is coming from an author who isn't `data.me`. Otherwise, special checks should be done when generating this GitHub url.

Reply:
Ok, fixed.